src/Approximations/overapproximate.jl

```diff
@@ -1636,6 +1687,74 @@ function overapproximate(X::LazySet{N}, ::Type{<:Zonotope},
     return Zonotope(c, G)
 end
 
+# overload on direction type
+function _overapproximate_zonotope_vrep(X::LazySet{N},
+                                        dir::Type{<:AbstractDirections{N}};
+                                        solver=default_lp_solver(N)) where {N}
+    n = dim(X)
+    return _overapproximate_zonotope_vrep(X, dir(2), solver=solver)
```

Comment:
```suggestion
    return _overapproximate_zonotope_vrep(X, dir(dim(X)), solver=solver)
```

Reply:
Note that these methods dispatching on `dir::Type{<:AbstractDirections{N}}` are never used. Should we rather remove them?

Reply:
i sometimes use them from outside

Reply:
consider it a feature :)

Reply:
Then maybe they should be tested. This one was broken.

Reply:
Those were done to allow the user to do something like `overapproximate(X, Zonotope, OctDirections)` (without specifying the dimensionality)

Reply:
But they are not used in that case. You call these helper methods with the instantiated directions only from inside the library. That is why this error was not detected.